crates/re_data_ui/src/app_id.rs

```diff
@@ -51,5 +53,50 @@ impl crate::DataUi for ApplicationId {
                 }
             });
         }
+
+        // ---------------------------------------------------------------------
+
+        ui.add_space(8.0);
+
+        // ---------------------------------------------------------------------
+
+        // Blueprint section.
+        let active_blueprint = ctx.store_context.blueprint;
+        let default_blueprint = ctx.store_context.hub.default_blueprint_for_app(self);
+
+        let button = egui::Button::image_and_text(
+            re_ui::icons::RESET.as_image(),
+            "Reset to default blueprint",
+        );
+
+        let is_same_as_default = default_blueprint.map_or(false, |default_blueprint| {
+            default_blueprint.latest_row_id() == active_blueprint.latest_row_id()
+        });
+
+        if is_same_as_default {
+            ui.add_enabled(false, button)
+                .on_disabled_hover_text("No modifications has been made");
+        } else if default_blueprint.is_none() {
+            ui.add_enabled(false, button)
+                .on_disabled_hover_text("There's no default blueprint");
+        } else {
+            // The active blueprint is different from the default blueprint
+            if ui
+                .add(button)
+                .on_hover_text("Reset to the default blueprint for this app")
+                .clicked()
+            {
+                ctx.command_sender
+                    .send_system(re_viewer_context::SystemCommand::ClearActiveBlueprint);
+            }
+        }
+
+        if ui.add(egui::Button::image_and_text(
+                re_ui::icons::RESET.as_image(),
+                "Reset active and default blueprint",
```

Comment:
This is such an annoying one to describe concisely... I think I find something like this more clear and parallels the above.
```suggestion
                "Reset to heuristic blueprint",
```